formats/financial_releases_campaign/frontend/examples/financial_releases_campaign.json

```diff
@@ -0,0 +1,20 @@
+{
+  "content_id": "333e6c9d-aaf9-4f7d-98ce-3ce57a79802e",
+  "base_path": "/lloydsshares",
+  "title": "Financial releases campaign page",
+  "description": "Campaign page for financial press releases",
+  "locale": "en",
+  "public_updated_at": "2015-12-07T11:22:21.000Z",
+  "format": "financial_releases_campaign",
+  "details": {
+    "image": {
+      "alt_text": "HM Government",
+      "url": "campaign.large.jpg"
+    },
+    "header_legal_disclaimer": "Header legal disclaimer Vel ut iudico dolore. Vix ne diam iuvaret referrentur, eam nisl accusamus temporibus ne. Nec quot deserunt dissentiet ut, ad  be available from <a href='/lloydsshares'>www.gov.uk/lloydsshares</a>.",
+    "title": "An Campaign Title",
+    "content": "<p>Vis nostro intellegebat at. Impedit pericula duo ne, pri nonumes ullamcorper in. Oratio invidunt et his, ele, at principes similique nam. No eam eleifend euripidis intellegat, sensibus theophrastus consectetuer.</p><p>Sumo solum quaestio has cu. Eu oblique euismod deleniti eum, congue necessitatibus ne sit.</p><p>No eum mutat soleat definiebas, ut argumentum efficiantur delicatissimi ius. At feugait adversarium vel. At cum mutat numquam graecis. Sit appareat adolescens constituto eu, te eleifend postulant theophrastus eam</p><p>Nulla regione quaeque ei pro, pro vulputate temporibus efficiantur id. Sit eu solum mediocrem, est te pertinax suavitate temporibus, et est incorrupte comprehensam.</p><p>Ne patrioque democritum mea, per natum doctus eu. Vel eu doming iudicabit. Al probo erant euripidis nec, blandit singulis euripidis an vel.</p><p>If you would like to receive email updates for daily ice cream flavours, send us a hand written note.</p><p>Vel ut iudico dolore. Vix ne diam iuvaret referrentur, eam nisl accusamus temporibus ne. If you like yoga <a id='lloyds-factsheet' href='/lloydsfactsheet' target='_blank'>click here</a></p>",
```

Comment:
Looking at the other schemas, I think `body` would be a more consistent name for this, unless I've misunderstood?